app/views/guests/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Guests <%= @guest.name %></h1>
+<hr>
+<div id="rooms">
+  <b>Guest <%= @guest.name %>'s Rooms:</b><br>
+  <% @guest.rooms.each do |room| %>
+  <ul> 
+  <li>Suite:<%= room.name %><li><br>
+  <li>Rate:<%= room.rate %><li><br>
+  <%# <li>Name:<%= room.hotel.name %><li><br> %>
+  <% end %>
+  </ul>
+
+  <div id="add-room">
+  <%= form_with model: @guest, local: true do |form| %>
```

Comment:
This wouldn't point to the `@guest` model because we want to create an association between a Guest and a Room.

```ruby
<%= form_with url: "/guests/#{@guest.id}/rooms", local: true do |form| %>
```

or 

```ruby
<%= form_with url: "/guests_rooms", local: true do |form| %>
    <%= form.hidden :guest_id, value: @guest.id %>
```